frontend/packages/dev-console/integration-tests/features/virtulization/virtualMachines-template.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Create Virtual Machine templates
+    As a user, I should be able create virutal machine template to create virtual machine
+
+
+    Background:
+        Given user has installed OpenShift Virtualization Operator
+        And user has created the OpenShift Virtualization Deployment
+        And user has created the HostPathProvisioner Deployment
+        And user is at administrator perspective
+        And user has selected namespace "aut-virtualization"
+        
+
+
+    @smoke, @cnv, @regression
+    Scenario: Create Virutal Machine template
+        Given user is at Virtualization tab under Workloads nav-item
```

Comment:
I think this is missing: user is at the administration view

Reply:
It's in the background